tools/resourcedocsgen/cmd/metadata.go

```diff
@@ -119,17 +119,15 @@ func PackageMetadataCmd() *cobra.Command {
 				mainSpec.Repository = "https://github.com/" + repoSlug.String()
 			default:
 				r, err := url.Parse(mainSpec.Repository)
-				if err == nil {
-					err = repoSlug.Set(r.Path)
-				}
 				if err != nil {
 					return errors.Wrapf(err, "parsing repo url %q from schema", mainSpec.Repository)
 				}
+
 				if r.Hostname() != "github.com" {
 					return fmt.Errorf("mainSpec.Repository host must be from GH, found %q", r.Hostname())
 				}
 
-				if err := repoSlug.Set(r.Path); err != nil {
+				if err := repoSlug.Set(strings.Trim(r.Path, "/")); err != nil {
```

Comment:
`r.Path` takes the form `/pulumi/pulumi-example`, but `repoSlug.Set` handles `pulumi/pulumi-example`.

Reply:
Is the bug that https://github.com/pulumi/registry/blob/e461135ff8e5556ed1265371a980b89d30ec7ddd/tools/resourcedocsgen/cmd/metadata.go#L464 expects there to be only one `/` in the slug?

Can we make a comment to this effect?

Reply:
Done.